avocado/core/mux.py

```diff
@@ -109,26 +109,21 @@ def update_defaults(self, defaults):
         combination.merge(self.root)
         self.variants = MuxTree(combination)
 
-    def str_variants(self):
+    def str_variants(self, summary=False, contents=False):
```

Comment:
Yep, this patch was intended for squash, if accepted (the problem is again that this PR contains several stages, because previous stages were not reviewed until I finished those new parts).